hu/dashboard/account.php

```diff
@@ -2,27 +2,27 @@
 
 return [
     'email' => [
-        'title' => 'Update your email',
-        'updated' => 'Your email address has been updated.',
+        'title' => 'Email frissítése',
+        'updated' => 'Az email címed sikeresen frissítve.',
     ],
     'password' => [
-        'title' => 'Change your password',
-        'requirements' => 'Your new password should be at least 8 characters in length.',
-        'updated' => 'Your password has been updated.',
+        'title' => 'Jelszó megváltoztatása',
+        'requirements' => 'Az új jelszavadnak legalább 8 karakter hosszúnak kell lennie.',
+        'updated' => 'A jelszavad sikeresen frissítve.',
     ],
     'two_factor' => [
-        'button' => 'Configure 2-Factor Authentication',
-        'disabled' => 'Two-factor authentication has been disabled on your account. You will no longer be prompted to provide a token when logging in.',
-        'enabled' => 'Two-factor authentication has been enabled on your account! From now on, when logging in, you will be required to provide the code generated by your device.',
-        'invalid' => 'The token provided was invalid.',
+        'button' => '2-Faktoros hitelesítés konfigurálása',
+        'disabled' => 'A 2-Faktoros hitelesítő ki van kapcsolva a fiókodon. A bejelentkezéskor a továbbiakban nem lesz szükség token megadására.',
+        'enabled' => 'A 2-Faktoros hitelesítő engedélyezve a fiókodon! Mostantól a bejelentkezéskor meg kell adnod az eszközöd által generált kódot.',
+        'invalid' => 'A megadott token érvénytelen.',
```

Comment:
Simán lehet "kétfaktoros"
```suggestion
        'button' => 'Kétfaktoros hitelesítés konfigurálása',
        'disabled' => 'A kétfaktoros hitelesítő ki van kapcsolva a fiókodon. A bejelentkezéskor a továbbiakban nem lesz szükség token megadására.',
        'enabled' => 'A kétfaktoros hitelesítő engedélyezve a fiókodon! Mostantól a bejelentkezéskor meg kell adnod az eszközöd által generált kódot.',
        'invalid' => 'A megadott token érvénytelen.',
```